test/test_linsolve.jl

```diff
@@ -1,7 +1,6 @@
 @eval module $(gensym())
 using ITensors: ITensors, MPO, OpSum, apply, randomMPS, siteinds
-using ITensorTDVP: ITensorTDVP, dmrg
-using KrylovKit: linsolve
+using ITensorTDVP: ITensorTDVP, dmrg, linsolve
```

Comment:
I prefer the previous style of `using KrylovKit: linsolve` which makes it clearer where that function originates from.